contexts/utopia-citizenship-v1.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "CompleteMrzBarcode": "https://w3id.org/citizenship/utopia#CompleteMrzBarcode",
```

Comment:
This is problematic and likely should be using `/utopia/citizenship#` for improved scope of utopia specific data. Leaving as is for now due to test data reasons.